app.rb

```diff
@@ -0,0 +1,100 @@
+require_relative 'helper'
+
+class App
+  def initialize
+    @labels = LabelModule.new
+    @authors = AuthorModule.new
+    @genres = GenreModule.new
+    @books = BookModule.new(@labels, @authors, @genres)
+    @games = GameModule.new(@labels, @authors, @genres)
+    @music_albums = MusicAlbumModule.new(@labels, @authors, @genres)
+  end
+
+  attr_accessor :books
+
+  def user_menu
+    puts ' '
+    puts 'Welcome to the Catalog of Things'
+    puts "Please Chose the following options:
+                    1. List all books
+                    2. List all music albums
+                    3. List all movies
+                    4. List of games
+                    5. List all genres
+                    6. List all labels
+                    7. List all authors
+                    8. List all sources
+                    9. Add a book
+                    10. Add a music album
+                    11. Add a movie
+                    12. Add a game
+                    13. Leave the catalog"
+    puts ' '
+  end
+
+  def user_choice(user_input)
+    @options = [
+      { user_choice: 1, action: :list_books },
+      { user_choice: 2, action: :list_all_music_albums },
+      { user_choice: 3, action: :list_all_movies },
+      { user_choice: 4, action: :list_all_games },
+      { user_choice: 5, action: :list_all_genres },
+      { user_choice: 6, action: :list_all_labels },
+      { user_choice: 7, action: :list_all_authors },
+      { user_choice: 8, action: :list_all_sources },
+      { user_choice: 9, action: :add_a_book },
+      { user_choice: 10, action: :add_a_music_album },
+      { user_choice: 11, action: :add_a_movie },
```

Comment:
- [ ] I couldn't add a movie because of the error I got, the error says the `add_a_movie` function is undefined. Since this requirement is for a group of 4 people, I recommend you remove it.

![cati](https://user-images.githubusercontent.com/53124548/219372554-f80d3094-a913-4a5d-96d2-de4c07e72fd9.png)

Reply:
@elisha2kyakpo1 it didn't work because as you can see we are a group of three, not four, and as the requirements said all of the listed options should appear on the home page

Reply:
Hi @Cluab, I'm sorry for this, I adverse you to remove the options not implemented. It's pointless to have such options when they are not working. At some point, you will share this app with a recruiter and it will not be a good thing if certain options are throwing errors.

Reply:
resolved! ✔️